yarn-project/cli/src/index.ts

```diff
@@ -14,11 +14,11 @@ import { JsonStringify } from '@aztec/foundation/json-rpc';
 import { DebugLogger, LogFn } from '@aztec/foundation/log';
 import { fileURLToPath } from '@aztec/foundation/url';
 import { compileContract, generateNoirInterface, generateTypescriptInterface } from '@aztec/noir-compiler/cli';
+import { createLibP2PPeerId } from '@aztec/p2p';
```

Comment:
Maybe we can instead just depend on `libp2p/peer-id-factory` directly, so we don't have to bring the whole libp2p into the CLI? It's not too bad though, the CLI is a devtool after all and it's ok if it's a bit bloated.

Reply:
Yes, good idea. Done